Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -367,10 +360,17 @@ public enum TextfieldType: String {
             self.setURLStackView(hiddenStatus: true)
         }
         
+        //set username and password fields status (enabled/disabled)
```

Comment:
This could be extracted in a method with the `enabledEditUrlUsernamePassword` passed as parameter. If you do that you can abstract the logic and also you avoid the need of write a comment to explain what this code does. I think this would lead us into a more cleaner code IMHO.